tests/ctst/Dockerfile

```diff
@@ -3,6 +3,8 @@ ARG CTST_TAG
 FROM registry.scality.com/sorbet/sorbet:$SORBET_TAG as sorbet
 FROM registry.scality.com/cli-testing/cli-testing:$CTST_TAG
 
+VOLUME /artifacts
```

Comment:
i am not sure we _want_ to create a volume here: which would create lead to automatically creating "anonymous" volume if run without binding/... We can create a volume bind even if the image does not have such a volume, so IMHO we should just leave the image as ebfore (esp. since the `/artifacts` path is a parameter we specify when running the tests)